site/docs/foundations/size.mdx

```diff
@@ -98,6 +99,10 @@ Bar is used to control the thickness of an accent bar, either in vertical or hor
   label="Show sizes"
 />
 
+## Indicator
```

Comment:
Slight curious on the order of the tokens. In the table above, this is the last one. In code, we do alphabetical?

Reply:
thanks, I need to update the table. Paul defined the order in his other comment